internal/configuration/settings/openvpnselection.go

```diff
@@ -43,6 +43,7 @@ func (o OpenVPNSelection) validate(vpnProvider string) (err error) {
 		providers.Ipvanish,
 		providers.Perfectprivacy,
 		providers.Privado,
+		providers.SlickVPN,
```

Comment:
@qdm12 document settings validation might need to be changed when adding a provider.

Reply:
Done locally, will push soon.